examples/tree_2d_dgsem/elixir_euler_kelvin_helmholtz_instability_amr_neuralnetwork_perssonperaire.jl

```diff
@@ -1,8 +1,9 @@
 using Downloads: download
 using Flux
+using Random
```

Comment:
Why do we need `Random` here? Note that we have `using Random: seed!` below.

Reply:
Fixed in b53c60e1.